Makefile

```diff
@@ -167,10 +209,27 @@ update-ui:
 	@printf $(COLOR) "Install/update temporal ui-server..."
 	@go install github.com/temporalio/ui-server/cmd/server@latest
 
-update-tools: update-goimports update-linters update-mockgen update-proto-plugins update-proto-linters update-gotestsum
+$(TEMPDIR) := $(shell mktemp -d)
+
+# go-install-tool will 'go install' any package with custom target and name of binary, if it doesn't exist
+# $1 - target path with name of binary (ideally with version)
+# $2 - package url which can be installed
+# $3 - specific version of package
+# This is courtesy of https://github.com/kubernetes-sigs/kubebuilder/pull/3718
+define go-install-tool
+@[ -f $(1) ] || { \
+set -e; \
+package=$(2)@$(3) ;\
+printf $(COLOR) "Downloading $${package}" ;\
+GOBIN=$(shell pwd)/$(TEMPDIR) go install $${package} ;\
+mv "$$(echo "$(1)" | sed "s/-$(3)$$//")" $(1) ;\
+}
+endef
+
+#update-tools: update-goimports $(GOLANGCI_LINT) $(MOCKGEN) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_HELPERS) $(PROTOGEN) update-proto-plugins update-proto-linters $(GOTESTSUM)
 
 # update-linters is not included because in CI linters are run by github actions.
-ci-update-tools: update-goimports update-mockgen update-proto-plugins update-proto-linters update-gotestsum
+#ci-update-tools: $(GOIMPORTS_BIN) $(MOCKGEN) update-proto-plugins update-proto-linters $(GOTESTSUM)
```

Comment:
You're going to remove this comment, right?

Reply:
Yep. I did that to test things